src/FileSearchApplication.java

```diff
@@ -0,0 +1,51 @@
+/*It is the main class where I am taking the filepath as an input from user and checking the presence of seached word*/
+
+import java.sql.SQLException;
+import java.util.StringTokenizer;
+
+public class FileSearchApplication {
+    protected static String contentOfFile, filepath, searchedWord;
```

Comment:
Please don't use static variables. Use local variable and function parameters when needed.